connector_prestashop/models/delivery.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from openerp import models, fields
+_logger = logging.getLogger(__name__)
+
+
+class PrestashopDeliveryCarrier(models.Model):
+    _name = 'prestashop.delivery.carrier'
+    _inherit = 'prestashop.binding'
+    _inherits = {'delivery.carrier': 'openerp_id'}
+    _description = 'PrestaShop Carrier'
+
+    openerp_id = fields.Many2one(
+        comodel_name='delivery.carrier',
+        string='Delivery carrier',
+        required=True,
+        ondelete='cascade'
+    )
+    id_reference = fields.Integer(
+        string='Id reference',
+        help="In PrestaShop, carriers with the same 'id_reference' are "
```

Comment:
In Prestashop, carriers can be copied with same 'Reference ID' (only the last copied carrier will be synchronized with the ERP)